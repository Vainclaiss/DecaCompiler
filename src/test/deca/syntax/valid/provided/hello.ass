; start main program
; Main program
	ADDSP #0
; Beginning of main instructions:
	WSTR "Hello"
	WNL
	HALT
; end main program
