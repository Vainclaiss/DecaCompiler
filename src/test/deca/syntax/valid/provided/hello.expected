// BEGIN LEXER //
OBRACE: [@0,136:136='{',<4>,10:0]
PRINTLN: [@1,146:152='println',<28>,11:8]
OPARENT: [@2,153:153='(',<6>,11:15]
STRING: [@3,154:160='"Hello"',<37>,11:16]
CPARENT: [@4,161:161=')',<7>,11:23]
SEMI: [@5,162:162=';',<8>,11:24]
CBRACE: [@6,164:164='}',<5>,12:0]
// END LEXER //